lib/widgets.js

```diff
@@ -43,6 +45,57 @@ var input = function (type) {
     };
 };
 
+var isString = function (value) {
+    return typeof value === 'string';
+};
+
+var isArray = Array.isArray;
```

Comment:
lol, i did not realize that. let's keep this here for now then, and i'll fix all of them later.